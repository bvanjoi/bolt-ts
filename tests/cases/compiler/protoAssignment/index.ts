// From `github.com/microsoft/TypeScript/blob/v5.7.3/tests/cases/compiler/protoAssignment.ts`, Apache-2.0 License

interface Number extends Comparable<number> {
  //~^ ERROR: Cannot find name 'Comparable'.

  compareTo(other: number);

}

Number.prototype.compareTo = function (other: number) {

 return this.valueOf() == other;

}


