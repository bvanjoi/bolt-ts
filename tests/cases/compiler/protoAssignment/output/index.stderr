  × Cannot find name 'Comparable'.
   ╭─[../../tests/cases/compiler/protoAssignment/index.ts:3:26]
 3 │ Comparable
   · ──────────
   ╰────
